helm_tests/other/test_redis.py

```diff
@@ -143,7 +143,9 @@ def test_redis_by_chart_password(self, executor):
         self.assert_broker_url_env(k8s_obj_by_key)
 
     @pytest.mark.parametrize("executor", CELERY_EXECUTORS_PARAMS)
-    def test_redis_by_chart_password_secret_name_missing_broker_url_secret_name(self, executor):
+    def test_redis_by_chart_password_secret_name_missing_broker_url_secret_name_and_broker_url_cmd(
```

Comment:
add some test when you have `AIRFLOW__CELERY__BROKER_URL_CMD`

Reply:
@romsharon98 too late, sorry.
i pushed simple unit test!